.travis.yml

```diff
@@ -7,4 +7,3 @@ with_content_shell: false
 dart_task:
 - test: --platform vm
 - dartanalyzer: --fatal-warnings lib
-- dartfmt
```

Comment:
I've noticed that the Dart 1 and Dart 2 formatters format slightly differently. Is that why this was taken out?